effekt/jvm/src/main/scala/effekt/Runner.scala

```diff
@@ -321,6 +321,9 @@ object LLVMRunner extends Runner[String] {
     val llPath  = basePath + ".ll"
     val optPath = basePath + ".opt.ll"
     val objPath = basePath + ".o"
+    val linkedLibraries = Seq(
+      "-lm", // Math library
+    ) ++ libuvArgs
```

Comment:
cc @phischu, do you foresee any problems with linking in `-lm` in the LLVM backend?
This is only relevant when using `gcc`, it works fine without it on `clang`.

Reply:
Not an actual problem, just that this is the beginning of a never ending stream of C libraries we are linking against.